src/sentry/static/sentry/app/utils.tsx

```diff
@@ -184,37 +180,42 @@ export function getShortVersion(version) {
   return version;
 }
 
-export function parseRepo(repo) {
-  if (!repo) {
-    return repo;
-  } else {
+export function parseRepo<T>(repo: T): T {
+  if (typeof repo === 'string') {
     const re = /(?:github\.com|bitbucket\.org)\/([^\/]+\/[^\/]+)/i;
     const match = repo.match(re);
     const parsedRepo = match ? match[1] : repo;
-    return parsedRepo;
+    return parsedRepo as any;
   }
+
+  return repo;
 }
 
 /**
  * Converts a multi-line textarea input value into an array,
  * eliminating empty lines
  */
-export function extractMultilineFields(value) {
+export function extractMultilineFields(value: string): Array<string> {
   return value
     .split('\n')
     .map(f => trim(f))
     .filter(f => f !== '');
 }
 
-function projectDisplayCompare(a, b) {
+interface Project {
```

Comment:
Can we use https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/types/index.tsx#L8?